src/components/text/ThemeText.tsx

```diff
@@ -12,7 +12,7 @@ import {
 } from '@atb/theme/colors';
 import {ContrastColor} from '@atb-as/theme';
 
-type ColorType = TextColor | StaticColor | ContrastColor;
+export type ColorType = TextColor | StaticColor | ContrastColor;
```

Comment:
Maybe not export this, and just use `TextColor` directly in `TransportModes`? The `ColorType` is probably broader than what we want `TransportModes` to support, and includes stuff like background themes (like `background_accent_1`) and message box status colors (like `info` and `warning`).